src/lib/wallet/GoodWallet.js

```diff
@@ -454,15 +464,27 @@ export class GoodWallet {
     return sha3(otlCode)
   }
 
+  /**
+   * checks against oneTimeLink contract, if the specified link has already been used or not.
+   * @param {string} link
+   */
   async isWithdrawLinkUsed(link: string) {
     const { isLinkUsed } = this.oneTimePaymentLinksContract.methods
     return await isLinkUsed(link).call()
   }
 
+  /**
+   * Checks if getWithdrawAvailablePayment returned a valid payment (BN handle)
+   * @param {*} payment
```

Comment:
Here I think that we should change the type from `{*}` to `{BN}`

And `isWithdrawPaymentAvailable(payment: any)` to `isWithdrawPaymentAvailable(payment: typeof BN)`